File: MSAU_Spring/msau-anboard/src/main/java/com/msau/onboard/entity/OnBoardee.java
package com.msau.onboard.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OnBoardee {
    private int demandId;
    private String fname;
    private String lname;
    private String startDate;
    private String bgcStatus;
    private int managerId;
    private String location;
    private String etaCompletion;
    private String email;
    private String dob;
    private String onboardStatus;
    MSManager manager;
    Skills skills;
}

File: MSAU_Angular/msau-onboarding/src/app/onboardees/models/onboardee.ts
export interface OnboardeeModel {
    fname: string,
    lname: string,
    email: string,
    phone: number,
    dob: string,
    startDate: string,
    location: string,
    managerId: number
    bgcStatus: string
    onboardStatus: string,
    etaCompletion: string
    skills: {
        skillList: Array<any>
    }
}
